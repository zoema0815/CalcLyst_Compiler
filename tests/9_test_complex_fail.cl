int main () {
    Imaginary b;
    Imaginary c;
    Complex x;
    b = 3.2i;
    c = b;
    c = 1;
    return 0;
}