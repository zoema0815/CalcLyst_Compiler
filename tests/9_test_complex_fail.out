Fatal error: exception Failure("illegal assignment Imaginary = int in c = 1")
